### PR TITLE
(#184) Create list component

### LINKED ISSUE #184: Build: List Component
### Build List component based on design specs/requirements documented in this ticket

> **ESTIMATE** 3

### Variants Information:
* Related USWDS Links: https://designsystem.digital.gov/components/list/
* Variants: ordered, unordered, unstyled

### Invision Mockups: 
  * Desktop: N/A
  * Tablet: N/A
  * Mobile: N/A

### Notion Documentation:
 * N/A

---

### Tasks
- [ ] Update Documentation Site (docs)
  - [ ] Create List component that includes:
    - Developer Documentation
      -  What SASS partial to import
      - (HTML Tab) What is the HTML?
    - **IA Provided** Guidance Documentation  
      - Includes when/how the component should be used - copied from USWDS
- [ ] Update NCIDS-CSS
  - list

### PR DESCRIPTION
**Pull request by olitharp-nci**
Closes #184

### CODE REVIEW

**Review by karlikpj — APPROVED** [review]

File: docs/src/components/list/list.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+				<ul className={listClass}>{listItems}</ul>
+			)}
+		</>
+	);
+};
+
+List.propTypes = {
+	children: PropTypes.node,
+	component: PropTypes.string,
+	variant: PropTypes.string,
```
Comment (arcepaul) [inline docs/src/components/list/list.jsx:33]: treat as an enum, (e.g. `PropTypes.oneOf(['unordered', 'ordered', unstyled])` )

**Review by karlikpj — APPROVED** [review]

File: docs/src/components/list/list.jsx (hunk 1 above)
Comment (olitharp-nci) [reply docs/src/components/list/list.jsx]: Updated

**Comment by arcepaul** [comment]
Looks like we need to update the :visited styling on the footer links

Screen Shot 2021-07-12 at 1 54 28 PM

**Comment by olitharp-nci** [comment]
> Looks like we need to update the :visited styling on the footer links
> 
> Screen Shot 2021-07-12 at 1 54 28 PM

@arcepaul updated

File: docs/src/components/footer.jsx [hunk 2]
```diff
@@ -15,18 +18,30 @@ const Footer = () => (
...
+						</List>
 					</div>
 					<div className="tablet:grid-col">
-						<ul className="add-list-reset">
-							<li>Footer link</li>
-							<li>Footer link</li>
-							<li>Footer link</li>
-						</ul>
+						<List variant="unstyled" component="footer">
+							<Link to="/#" title="Link 1" aria-label="Link A">
```
Comment (arcepaul) [inline docs/src/components/footer.jsx:35]: I don't think we need title and label here as they are plain links

**Comment by olitharp-nci** [comment]
Closing due to changing component tickets

**Pull request closed without merging**